crates/next-api/src/module_graph.rs

```diff
@@ -579,6 +708,31 @@ impl ReducedGraphs {
         .instrument(span)
         .await
     }
+
+    #[turbo_tasks::function]
+    pub async fn validate_pages_css_imports(
+        &self,
+        entry: Vc<Box<dyn Module>>,
+        app_module: Vc<Box<dyn Module>>,
+    ) -> Result<()> {
+        let span = tracing::info_span!("validate pages css imports");
```

Comment:
nit: drop the span? don't we get one implicitly from turbotasks?

Reply:
no, we don't

if you build locally, and thus have `NEXT_TURBOPACK_TRACING=turbo-task` at your disposal, then yes every turbo task has a span

but in the released version, the higher-granularity tracing modes are not available and the only spans you can see are the manually added ones (like this one)

Reply:
Thanks, i keep forgetting that.